Source/Eto.WinForms/Forms/WindowsControl.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Eto.WinForms.Forms
 {
-	public interface IWindowsControl
+	public interface IWindowsControl: Control.IHandler
```

Comment:
Added it that we can access `BackgroundColor` as well (all the controls implement `Control.IHandler` anyway)